tools/python/boutiques/localExec.py

```diff
@@ -435,7 +435,24 @@ def _localExecute(self, command):
             sys.stderr.write('Input Value Error during attempted execution!')
             raise e
         else:
```

Comment:
Style: I don't think that `else` here is necessary, since the exceptions also raise exceptions. Removing `else` would save one level of indentation.